### ISSUE TITLE
Starship Troopers: Terran Command (1202130)

### ISSUE THREAD

**Issue by LiamDawe** [ValveSoftware/Proton#5916]
# Compatibility Report
- Name of the game with compatibility issues: Starship Troopers: Terran Command 
- Steam AppID of the game: 1202130

## System Information
- GPU: nvidia 2080ti
- Driver/LLVM version: 510.68.02
- Kernel version: 5.17-13
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/LiamDawe/5667c3adbdbf1ff22252e0cfff577202
- Proton version: 7.0-3

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 

The videos do not work. They work with GE-Proton but they're completely discoloured needing `WINE_DO_NOT_CREATE_DXGI_DEVICE_MANAGER=1 %command%`

Discoloured example in GE:
![Screenshot_20220616_184511](https://user-images.githubusercontent.com/83223/174136858-0a7d7858-874d-4eab-9b17-3157ce108f9a.png)

## Reproduction

Just try to run it on Proton 7 or Experimental, see the videos don't work. Try on GE where the main videos work but even with using the fix noted above, some of the in-mission video help is clearly not correctly showing:
![Screenshot_20220616_185834](https://user-images.githubusercontent.com/83223/174136890-b152498b-d6c7-46c9-bed5-65d1bc9fbc12.png)
The scaling is completely off there

**Comment by ghtesting2020** [comment]
Thank you @kisak-valve for the launch option WINE_DO_NOT_CREATE_DXGI_DEVICE_MANAGER=1 %command% that fixed the rainbow colors for me. Can I ask how you figure this kind of stuff out? There's a lot of games I can't even get to run that are Gold in protondb.

**Comment by LiamDawe** [comment]
You tagged the wrong person. I found it because I asked in a Discord.

**Comment by ghtesting2020** [comment]
> You tagged the wrong person. I found it because I asked in a Discord.

you're right, my bad. Thanks goes to you instead! Is the discord you mentioned for proton gaming or something? I need some masters to help me out, I am going crazy with some of these games hahahaa.

**Comment by alasky17** [comment]
@LiamDawe  @ghtesting2020 I believe that WINE_DO_NOT_CREATE_DXGI_DEVICE_MANAGER=1 %command% should no longer be necessary for this game with current experimental-7.0-20220629.  Could you confirm if this is the case?
